src/common/components/InlineList/InlineList.jsx

```diff
@@ -35,22 +38,24 @@ class InlineList extends Component {
 }
 
 InlineList.propTypes = {
-  items: PropTypes.oneOfType([PropTypes.instanceOf(List), PropTypes.array]),
-  renderItem: PropTypes.func.isRequired,
   extractKey: PropTypes.func,
-  suffix: PropTypes.node,
+  items: PropTypes.oneOfType([PropTypes.instanceOf(List), PropTypes.array]),
   label: PropTypes.string,
+  renderItem: PropTypes.func.isRequired,
   separateItems: PropTypes.bool,
+  separateItemsClassName: PropTypes.string,
+  suffix: PropTypes.node,
   wrapperClassName: PropTypes.string,
 };
 
 InlineList.defaultProps = {
-  label: null,
+  extractKey: item => item,
   items: null,
+  label: null,
+  separateItems: true,
+  separateItemsClassName: SEPARATE_ITEMS_CLASS,
```

Comment:
let's name this `DEFAULT_SEPARATE_ITEMS_CLASS` and change the name to `separate-items-with-semicolon`